doc/examples.md

```diff
@@ -0,0 +1,108 @@
+# Examples: Getting Started using Padertorch
+
+Padertorch's main purpose is to provide an open interface that simplifies tedious work and works with as many different 
+workflows as possible. Due to the open design of Padertorch, there is no "right" or "wrong" way to train a neural network.
+However, to make getting started easier, we have provided several examples with our recommended workflow when using Padertorch. 
+
+We recommend the usage of the following components when using Padertorch:
+  - [sacred](): Experiment Organization / Ensuring reproducability
+  - [lazy_dataset](https://github.com/fgnt/lazy_dataset): Data preparation and on-the-fly data generation
+  - [Padertorch Configurable](): Adjustment of training hyperparameters
```

Comment:
I think Configurable should not be its own point more sacred / pt.Configurable

Reply:
I think `sacred` and `Configurable` address different things.
But I agree, that the mentioned points are the shared things:
 - Configurable: Increase reproducibility, because it fills the config with the default values
   - I am not sure, if sacred has for us a relevant effect for the reproducibility. It is more logging.
 - sacred: Provide cli interface to change hyperparameters
 - Configurable: Provide advanced options to provide dependent hyperparameters.